android/app/src/main/java/org/ZingoLabs/Zingo/RPCModule.kt

```diff
@@ -58,7 +58,7 @@ class RPCModule internal constructor(private val reactContext: ReactApplicationC
         RustFFI.initlogging()
 
         // Create a seed
-        val seed = RustFFI.initnew(server, reactContext.applicationContext.filesDir.absolutePath, chainhint)
+        val seed = RustFFI.initnew(server, reactContext.applicationContext.filesDir.absolutePath, chainhint, "true")
```

Comment:
This needs to be reviewed by @juanky201271 and/or @fluidvanadium , and tested with e2e/manually...    I find it curious that the type of the mempool monitor state selector passed to `initnew` seems to be a String instead of a bool.   I will review the `RustFFI.initnew` definition to understand the available types in that context.

Reply:
thanks i will implement a bool to the FFI

Reply:
i will implement this in a new PR as it is a bit more involved than i expected